cmd/api/src/api/v2/auth/auth.go

```diff
@@ -495,7 +495,7 @@ func (s ManagementResource) updateUser(response http.ResponseWriter, request *ht
 
 func (s ManagementResource) ensureUserHasNoAuthSecret(context ctx.Context, user model.User) error {
```

Comment:
This function wasn't using the `ctx.Context` before anyway, so I'd prefer us to change the signature to `context.Context` to avoid the unnecessary `context.ConstructGoContext()`